sortthread.go

```diff
@@ -259,7 +259,7 @@ func (m *Mailbox) orderedSubjThread(tx *sql.Tx, uid bool, seqSet *imap.SeqSet, m
 			id := msg.id
 			if !uid {
 				var ok bool
-				id, ok = m.handle.UidAsSeq(id)
```

Comment:
Not sure what this change is about.

Reply:
Sorry, I forgot to comment right after creating the PR. My linter showed that this variable is not being used anywhere else. I can revert if it was marked wrongfully.

Reply:
Hmm, that's indeed seems to be a bug: `next.Id = msg.id` probably should be `next.Id = id`. Could you please change that?

Reply:
Done, Max.